internal/aws/proxy/cfg.go

```diff
@@ -38,6 +38,8 @@ type Config struct {
 	// will be called or not. Set to `true` to skip EC2 instance
 	// metadata check.
 	LocalMode bool `mapstructure:"local_mode"`
+
```

Comment:
Add some godoc to the field please

Reply:
Sorry, missed by mistake. I have added now